.github/workflows/references.yml

```diff
@@ -219,10 +219,10 @@ jobs:
           pip install -e .[torch,viz,html] --upgrade
       - if: matrix.framework == 'tensorflow'
         name: Evaluate text recognition (TF)
-        run: python references/recognition/evaluate_tensorflow.py crnn_mobilenet_v3_small --dataset IIIT5K -b 32
+        run: python references/recognition/evaluate_tensorflow.py crnn_mobilenet_v3_small --dataset SVT -b 32
```

Comment:
IIIT5K download url is down ..again .. wrote already an mail .. use another dataset for the CI test